pkg/pb/node.proto

```diff
@@ -80,3 +81,13 @@ message NodeRestrictions {
     int64 free_bandwidth = 1;
     int64 free_disk = 2;
 }
+
+// NodeVersion contains
+message NodeVersion {
+    int64 major = 1;
+    int64 minor = 2;
+    int64 patch = 3;
+    string hash = 4;
+    int64 timestamp = 5;
```

Comment:
can this be `google.protobuf.Timestamp timestamp = 5;`? you'll need to use https://godoc.org/github.com/golang/protobuf/ptypes#Timestamp and https://godoc.org/github.com/golang/protobuf/ptypes#TimestampProto for conversion

Reply:
done